test/ci/test_all_upgrades.bash

```diff
@@ -5,19 +5,8 @@ shopt -s failglob inherit_errexit
 
 project_root="$(dirname "$0")/../.."
 
-#
 # Versions/tags known to build
-#
-versions=(
-    '1.0.0'
-    '1.0.1'
-    '1.1.0'
-    '1.3.0'
-    '1.4.0'
-    '1.5.0'
-    '1.6.0'
-    '1.7.0'
-)
+mapfile -t versions < <(git tag --list '[0-9]*.[0-9]*.[0-9]*' | grep --fixed-strings --invert-match --line-regexp --regexp=1.2.0)
```

Comment:
Do we want to do all versions or only supported versions? What we have now is definitely wrong too, but maybe we only want to support a couple of versions that are actually run in production or run against supported versions of PostgreSQL?

Reply:
Good idea. Trading support of old versions for pipeline runtime would be worth it, at least if nobody (including third parties) is running them. Do you have a semi-definite list of versions you think we should support, or do you want to make this an issue to look into later?

Reply:
I think we should look into this later, so please raise another issue to improve this. I think the general maybe we should support major version - 2 for all supported PostgreSQL versions